tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -388,13 +390,22 @@ func (s *storeImpl) Upsert(ctx context.Context, obj *{{.Type}}) error {
     } else if !ok {
         return sac.ErrResourceAccessDenied
     }
-    {{- else if eq .ResourceType "globallyScoped" }}
+    {{- else if or (eq .ResourceType "globallyScoped") (eq .ResourceType "directlyScoped") }}
     scopeChecker := sac.GlobalAccessScopeChecker(ctx).AccessMode(storage.Access_READ_WRITE_ACCESS).Resource(resources.{{.Type | storageToResource}})
```

Comment:
```
scopeChecker := sac.GlobalAccessScopeChecker(ctx).AccessMode(storage.Access_READ_WRITE_ACCESS).Resource(targetResource)
```
(iterate for all functions)

Reply:
I'll do it in a separate PR

Reply:
https://github.com/stackrox/stackrox/pull/1283